recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,4 @@
-microarch_level:  # [x86_64]
-  - 1  # [x86_64]
+microarch_level:
+  - 1
   - 3  # [x86_64]

```

Comment:
I'm leaving this as

```yaml
- 3 # [x86_64]
```

and not the recommended

```yaml
- 3 # [unix and x86_64]
```

as I'm going to assume that you know what you're doing with your own feedstock. :+1:

Reply:
Yes, this package enables the AVX instructions with explicit CMake options that trigger the use of internal assembly implementations, so even if AVX is not enabled via compilers flags it is still useful to enable it.